### PR TITLE
Add basic stomp flow example

### PR DESCRIPTION
**Pull request by tomverran**
@patrickkerrigan this is what I was using to test the stomp client

You can either run it through SBT or with IntelliJ's "Application" run configuration that lets you pick a main class & fill in the required environment variables

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,10 @@
-*.class
-*.log
+dist/*
+target/
+lib_managed/
+src_managed/
```
Comment (tomverran) [inline .gitignore:4]: I always copy this from https://github.com/github/gitignore/blob/master/Global/SBT.gitignore

**Comment by tomverran** [comment]
👍 -did you get it to work locally?

**Comment by patrickkerrigan** [comment]
Yeah, it seems to be working! I certainly see the value in implementing the STOMP decoder as a stream now, that code is very clean

**Review by patrickkerrigan — APPROVED** [review]

**Comment by tomverran** [comment]
Indeed all the future parsing steps can probably just be pure code now called with map etc

**Pull request merged**